app/scripts/lib/metrics.js

```diff
@@ -66,22 +67,22 @@ define([
     this.timers = this._speedTrap.timers;
     this.events = this._speedTrap.events;
 
-    this._window = options.window || window;
+    this._window = Validate.isNotFalsy(options.window, window);
 
-    this._lang = options.lang || 'unknown';
-    this._context = options.context || 'web';
-    this._entrypoint = options.entrypoint || NOT_REPORTED_VALUE;
-    this._migration = options.migration || NOT_REPORTED_VALUE;
-    this._service = options.service || NOT_REPORTED_VALUE;
-    this._campaign = options.campaign || NOT_REPORTED_VALUE;
+    this._lang = Validate.isNotFalsy(options.lang, 'unknown');
```

Comment:
For cases like this where the function is used repeatedly, a few bytes can be shaved from the final output by saving a reference to the function into a local variable, and calling the local variable.

e.g.:
```js
var isNotFalsy = Validate.isNotFalsy;
....
this._lang = isNotFalsy(options.lang, 'unknown');
this._context = isNotFalsy(options.context, 'web');
...
```